src/db.c

```diff
@@ -645,6 +645,15 @@ robj *dbUnshareStringValue(redisDb *db, robj *key, robj *o) {
     return o;
 }
 
+static void clearHashtableOverhead(dict *d) {
+    if (dictIsRehashing(d)) {
+        server.overhead_hashtable_lut -= DICTHT_SIZE(d->ht_size_exp[1]);
+        server.overhead_hashtable_rehashing -= DICTHT_SIZE(d->ht_size_exp[0]);
```

Comment:
don't we need to deduct `exp[0]` from `_lut` too?
i.e. the `lut` metric counts both tables.